Resources/Prototypes/_CD/Entities/Clothing/OuterClothing/coats.yml

```diff
@@ -33,3 +33,130 @@
+  - type: Clothing
+    sprite: _CD/Clothing/OuterClothing/Coats/Europa/veterancoat.rsi
+  - type: TemperatureProtection
+    coefficient: 0.1
+  - type: Armor
+    modifiers:
+      coefficients:
+        Slash: 0.95
+        Heat: 0.90
+
+- type: entity
+  parent: ClothingOuterCoatEuropanVet
+  id: ClothingOuterCoatEuropanCoalition
+  name: coalition jacket
+  description: A heavy leather jacket, winterized and waterproofed for harsh underwater environments. Emblazoned with a Coalition patch on the shoulders.
+  components:
+  - type: Sprite
+    sprite: _CD/Clothing/OuterClothing/Coats/Europa/coalitioncoat.rsi
+  - type: Clothing
+    sprite: _CD/Clothing/OuterClothing/Coats/Europa/coalitioncoat.rsi
+
+- type: entity
+  parent: ClothingOuterCoatEuropanVet
+  id: ClothingOuterCoatEuropanSeparatist
+  name: separatist jacket
+  description: A heavy leather jacket, winterized and waterproofed for harsh underwater environments. Emblazoned with a Separatist patch on the shoulders.
+  components:
+  - type: Sprite
+    sprite: _CD/Clothing/OuterClothing/Coats/Europa/separatistcoat.rsi
+  - type: Clothing
+    sprite: _CD/Clothing/OuterClothing/Coats/Europa/separatistcoat.rsi
+
+# Cargo Gorka
+- type: entity
+  parent: ClothingOuterStorageFoldableBase
+  id: ClothingOuterCoatCargoBomber
+  name: cargo gorka
+  description: A heavy canvas gorka for cold shifts in Cargo.
+  components:
+  - type: Sprite
+    sprite: _CD/Clothing/OuterClothing/Coats/cargo_bomber.rsi
+  - type: Clothing
+    sprite: _CD/Clothing/OuterClothing/Coats/cargo_bomber.rsi
+  - type: TemperatureProtection
+    coefficient: 0.1
+
+- type: entity
+  parent: [ClothingOuterStorageFoldableBaseOpened, ClothingOuterCoatCargoBomber]
+  id: ClothingOuterCoatCargoBomberOpened
+  name: cargo gorka
+
+# CentCom Coat
+- type: entity
+  parent: ClothingOuterStorageBase
+  id: ClothingOuterCoatCentcomOfficer
+  name: CentCom officer's coat
+  description: A comfortable green coat worn by CentCom officers. The armour has been removed in favour of more pocket space.
+  components:
+  - type: Sprite
+    sprite: _CD/Clothing/OuterClothing/Coats/central_officer.rsi
+  - type: Clothing
+    sprite: _CD/Clothing/OuterClothing/Coats/central_officer.rsi
+
+# High-Vis jackets
+- type: entity
+  parent: ClothingOuterStorageFoldableBase
+  id: ClothingOuterCoatYellowHiVis
+  name: emergency jacket
+  description: A somewhat heavy jacket made of fluorescent fabric. This one is neon yellow.
+  components:
+  - type: Sprite
+    sprite: _CD/Clothing/OuterClothing/Coats/Highvis/yellow_bomber.rsi
+  - type: Clothing
+    sprite: _CD/Clothing/OuterClothing/Coats/Highvis/yellow_bomber.rsi
+  - type: TemperatureProtection
+    coefficient: 0.1
+
+- type: entity
+  parent: [ClothingOuterStorageFoldableBaseOpened, ClothingOuterCoatYellowHiVis]
+  id: ClothingOuterCoatYellowHiVisOpened
+  name: emergency jacket
+
+- type: entity
+  parent: ClothingOuterStorageFoldableBase
```

Comment:
Same as above

Reply:
Tempature protection for HiVis coats discussed here: https://discord.com/channels/1163352113162768436/1163352113859002400/1265796891187085334

It makes sense with the reference image.

Reply:
This still needs to inherit from ClothingOuterCoatYellowHiVis. Assuming you want the damage resistances.